### DISCUSSION TITLE
videojs flash player RTMP Streaming media playback screen stretch

### DISCUSSION THREAD

**Discussion by chaoshenghu** [bluenviron/mediamtx#929] [Questions]
Does the server send onMetaData data?
![image](https://user-images.githubusercontent.com/31982024/165431414-56375a68-26df-48ca-a1df-0c233f2605b8.png)

**Comment by aler9** [comment]
yes, it does:

https://github.com/aler9/rtsp-simple-server/blob/3e5f62156d3c442186c5b3b8ac4200d79d1e5cc2/internal/rtmp/conn.go#L324

[a comment by an automated account is not shown]
